tests/integration/components/bs-form-test.js

```diff
@@ -321,6 +321,46 @@ module('Integration | Component | bs-form', function (hooks) {
     assert.dom(`.${formFeedbackClass()}`).hasText('There is an error');
   });
 
+  test('Form with invalid validation shows validation errors immediately when @showValidations is true', async function (assert) {
```

Comment:
Would be great if you could extend the test to verify that success validation state is also shown.

Reply:
added test for success validation